app/posts/find-teacher-training/2022-02-11-updating-international-candidates-section.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Updating the ‘International candidates’ section
+description: Why we have updated and renamed this section
+date: 2022-02-11
+screenshots:
+  items:
+    - text: International candidates - sponsorship may be available
+      src: find-sponsorship-available.png
+    - text: International candidates - sponsorship not available
+      src: find-sponsorship-not-available.png
+related:
+  items:
+    - text: GitHub pull request making the change
+      href: https://github.com/DFE-Digital/find-teacher-training/pull/1138
+    - text: Design history entry for previous change
+      href: /find-teacher-training/new-international-students-section/
+---
+
+We previously added an [‘International students’ section](/find-teacher-training/new-international-students-section/) to help candidates from outside the UK get the information they need. This includes letting them know whether visa sponsorship is available for a course.
+
+We have renamed this section ‘International candidates’ because:
+
+* it's more consistent with ‘candidate’, which we use throughout Apply for teacher training
+* candidates may apply for a salaried course where the term ‘student’ is less applicable
+
+We have also updated the content to:
+
+* explicitly explain that candidates already have the right to work and study if they have ‘indefinite leave to remain’
+* mention other visa types which do not need sponsorship, such as Graduate and Family visas
+* link to GOV.UK guidance about [training to teach in England if you’re a non-UK citizen](https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen)
+
+
+Where the information from Publish indicates that the provider can sponsor the relevant visa for the course (Student or Skilled worker), we have made the guidance less certain:
+
+> Before you apply for this course, contact us to check [visa type] visa sponsorship is available.
+
+This is because visa sponsorship availability is more complicated than we previously believed, and depends upon either who the accrediting body is (for Student visas) or who the employer is (for Skilled worker visas), and therefore the information as collected through Publish may not be accurate.
```

Comment:
```suggestion
This is because providers cannot always guarantee that the accrediting body (for Student visas) or the employer (for Skilled worker visas) can sponsor visas. 
```

Reply:
@EmmaFrith I've made a tweak to this paragraph to try and summarise the situation more clearly. Let me know what you think!